packages/frontend-plugin-api/api-report.md

```diff
@@ -149,6 +169,29 @@ export interface ConfigurableExtensionDataRef<
   >;
 }
 
+// @public (undocumented)
+export type CoreBootErrorPageComponent = ComponentType<
+  PropsWithChildren<{
+    step: 'load-config' | 'load-chunk';
+    error: Error;
+  }>
+>;
+
+// @public (undocumented)
+export const coreBootErrorPageComponentRef: ComponentRef<CoreBootErrorPageComponent>;
```

Comment:
Thinking we could perhaps start by collecting these under a `coreComponentRefs` or something like that? Just to collect the exports a bit